pysoundcomparisons/__main__.py

```diff
@@ -264,25 +259,31 @@ def downloadSoundFiles(args, out_path=os.path.join(os.getcwd(), "sound"), db_nee
     if not os.path.exists(out_path):
         os.makedirs(out_path)
     for pth in sorted(desired_keys):
+        # check for stored extensions
+        available_ext = catalog.extensions_for_file_path_key(pth)
+        desired_ext_checked = list(set(available_ext) & set(desired_ext))
+        if len(desired_ext_checked) == 0:
+            if len(available_ext) > 0:
+                # fall back to first extension stored in catalog
+                desired_ext_checked.append(available_ext[0])
+            else:
+                print("ERROR - no sound file for key %s" % (pth), flush=True)
```

Comment:
The `args` argument passed into commands has a `log` attribute, which is a `Logger` object. So for this kind of output, we should rather use `args.log.warning(...)` or similar. The advantage of this is that filtering of these messages is simpler and can be done selectively using a logging configuration.

Reply:
ok